toolchains/cpu.spack.gyselalibxx_env/prepare.sh

```diff
@@ -45,6 +52,10 @@ git clone https://github.com/pdidev/spack.git spack-0.23.1/var/spack/repos/pdi
 spack repo add --scope site spack-0.23.1/var/spack/repos/pdi
 
 spack compiler find
+spack external find
+# after `spack external find`, un-find some packages that commonly cause issues (curl and perl)
+sed -i.save '/^\s*curl:/,/^\s*[^ ]/ s/^\(\s*\)/#\1/' ${SPACK_USER_CONFIG_PATH}/packages.yaml
+sed -i.save '/^\s*perl:/,/^\s*[^ ]/ s/^\(\s*\)/#\1/' ${SPACK_USER_CONFIG_PATH}/packages.yaml
```

Comment:
I am really not in favor of these lines. It seems to me a way to loose reproducibility between installations. Why do you need in the first place ?

Reply:
( I mean removing the whole `spack external find` thing)

Reply:
I am surprised to see this error, we did not have it in previous runs. I would be interested to understand what change could explain this error ?

Reply:
The error only appeared after the addition of `spack external find`